Data/Storable/Endian.hs

```diff
@@ -37,6 +38,12 @@ import GHC.Int
 newtype LittleEndian a = LE a
 newtype BigEndian    a = BE a
 
+getLittleEndian :: LittleEndian a -> a
```

Comment:
Thanks! I'm out-of-touch on Haskell style, but thanks for accepting this suggestion anyway.